volume/drivers/extpoint_test.go

```diff
@@ -21,3 +22,81 @@ func TestGetDriver(t *testing.T) {
 		t.Fatalf("Expected fake driver, got %s\n", d.Name())
 	}
 }
+
+// testParseMountSpec is a structure used by TestParseMountSpecSplit for
```

Comment:
Looking. I seem to recall there was a good reason for putting it here, but will verify this morning.